internal/rootcoord/task.go

```diff
@@ -324,6 +324,12 @@ func (t *DropCollectionReqTask) Execute(ctx context.Context) error {
 		t.core.ddlLock.Lock()
```

Comment:
should we put the t.core.chanTimeTick.addDdlTimeTick(ts, reason) at the very begining of the task?

I thought the main reason is not because the time consuming task, but we don't handle timetick correctly.